### PR TITLE
feat(editor): comment tags ui

### PR DESCRIPTION
**Pull request by Itrulia**
Screenshot 2022-08-25 at 14 21 48
Screenshot 2022-08-25 at 14 21 23

### CODE REVIEW

**Comment by Itrulia** [comment]
Soft deletes will come at a later time, ticket is already created @michael-scheurer

File: packages/editor/src/client/routes/tagList.tsx [hunk 1]
```diff
@@ -0,0 +1,330 @@
...
+      )}
+
+      <Form fluid formValue={formValue}>
+        {Object.keys(formValue).map(tagId => (
+          <FlexboxGrid key={tagId} style={{marginBottom: '12px'}}>
+            <div style={{maxWidth: '300px', flex: '1 1'}}>
+              <Form.Control
+                name={tagId}
+                placeholder={t('tags.overview.placeholder')}
+                onChange={(value: string) => {
```
Comment (michael-scheurer) [inline packages/editor/src/client/routes/tagList.tsx:241]: When user hits enter, tag should be saved imo.

File: packages/editor/src/client/routes/tagList.tsx [hunk 2]
```diff
@@ -0,0 +1,330 @@
...
+  | {type: TagListActionType.Delete; payload: {id: string}}
+
+const mapTagsToFormValue = (tags: Tag[] | null | undefined) =>
+  tags?.reduce((obj, node) => {
+    obj[node.id] = node.tag ?? ''
+
+    return obj
+  }, {} as Record<string, string>) ?? {}
+
+const tagFormValueReducer = (
```
Comment (michael-scheurer) [inline packages/editor/src/client/routes/tagList.tsx:53]: slightly too complicated? but fine for me.

**Review by michael-scheurer — APPROVED** [review]
Good work. See comments.

File: packages/editor/src/client/base.tsx [hunk 3]
```diff
@@ -126,13 +127,23 @@ export function Base({children}: BaseProps) {
...
                 </Nav.Item>
 
-                <Nav.Item
-                  as={NavLink}
-                  href="/comments"
-                  icon={<CommentIcon />}
-                  active={path === 'comments'}>
-                  {t('navbar.comments')}
-                </Nav.Item>
+                <Nav.Menu eventKey={'1'} title={t('navbar.comments')} icon={<CommentIcon />}>
```
[the remark this answers is not in the document]
Comment (Itrulia) [reply packages/editor/src/client/base.tsx:130]: Unfortunately could not figure out how to do that with rsuite without writing this myself, do you know how to properly do this?

**Pull request merged**